File: code-coaches/easy/halloween-candy/HalloweenCandy.java
import java.util.Scanner;

public class HalloweenCandy {
	
	public static void main(String[] args) {
		Scanner input = new Scanner(System.in);
		int houses;

		do {
			houses = input.nextInt();
		} while (houses < 3);

		int billProbability = (int) Math.ceil(100.0 / houses * 2);
		System.out.println(billProbability);

		input.close();
	}

}

File: code-coaches/easy/halloween-candy/halloween_candy.cpp
#include <iostream>
#include <cmath>

int main()
{
	int houses;

	do
	{
		std::cin >> houses;
	}
	while (houses < 3);

	int billProbability = ceil(100.0 / houses * 2);
	std::cout << billProbability;

	return 0;
}